pages/citizen/registration.py

```diff
@@ -51,6 +69,21 @@ def resend(self):
         click(self.ID.btnResend)
         return self
 
+    def enter_otp(self):
+        return get(self.ID.lblOTP)
+
+    def otp_sent_to(self):
+        return get(self.ID.lblOTPSentTo)
+
+    def otp_required(self):
+        return get(self.ID.lblErrorMsg)
+
+    def invalid_otp(self):
+        return get(self.ID.lblErrorMsg)
+
+    def otp_has_resent(self):
+        return get(self.ID.lblResentOTPMsg)
```

Comment:
Should use wait for toast and return that message

Reply:
Done